_locales/_untranslated_en.json

```diff
@@ -61,5 +61,14 @@
   },
   "later": {
     "message": "Later"
+  },
+  "no_stickers_yet": {
+    "message": "No stickers yet"
+  },
+  "add_stickers_instructions": {
+    "message": "Add some using the button below."
```

Comment:
not sure, how the button works, but i assume it just opens the folder?

i would then go for sth. like the following following message:

```
To add stickers, 
tap "Open Sticker Folder"
and drag image and sticker files to that folder.
```

to have less text, the error "No stickers yet" can also be removed, that seems obvious.

Reply:
addressed via https://github.com/deltachat/deltachat-desktop/pull/3293/commits/3ac757a5906cea2f105abdc37f7d3455cb3a8049